### PR TITLE
unify data process for inference

### PR DESCRIPTION
**Pull request by liangxhao**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

1. 增加识别模型的dynamic batch_size加载的支持
2. 更新部分数据预/后处理代码，基类继承于训练侧，将target_size从__init__挪到__call__中
3. 优化并行队列的数据存储逻辑
4. 评估脚本的数据格式随着训练侧更新

TODO: 补充测试用例

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Review by SamitHuang — APPROVED** [review]
lgtm

File: deploy/py_infer/src/configs/rec/ppocr/ch_PP-OCRv3_rec_distillation.yaml [hunk 1]
```diff
@@ -10,9 +10,10 @@ eval:
       - DecodeImage: # load image
           img_mode: BGR
           channel_first: False
-      - RecResizeImg:
-          padding: True
-      - NormalizeImage:
+      - RecResizeNormForInfer:
+          keep_ratio: False
```
Comment (VictorHe-1) [inline deploy/py_infer/src/configs/rec/ppocr/ch_PP-OCRv3_rec_distillation.yaml:14]: Todo: 其他yaml测试norm before pad

**Review by VictorHe-1 — APPROVED** [review]

File: deploy/py_infer/src/data_process/postprocess/det_postprocess.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+
+# add mindocr root path, and import postprocess from mindocr
+mindocr_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../../.."))
+sys.path.insert(0, mindocr_path)
+
+from mindocr.postprocess import det_db_postprocess
```
Comment (VictorHe-1) [inline deploy/py_infer/src/data_process/postprocess/det_postprocess.py:8]: 需要再import det_pse_postprocess

File: deploy/py_infer/src/data_process/postprocess/det_postprocess.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+__all__ = ["DBPostprocess"]
```
Comment (VictorHe-1) [inline deploy/py_infer/src/data_process/postprocess/det_postprocess.py:10]: PSEPostprocess同步加上

File: deploy/py_infer/src/data_process/preprocess/transforms/det_transforms.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class DetResize(det_transforms.DetResize):
+    # limit_type and force_divisable is not supported currently, because inference model don't support dynamic shape
+    def __init__(self,
+                 keep_ratio=True,
+                 padding=False,
+                 interpolation=cv2.INTER_LINEAR,
+                 **kwargs):
+        if keep_ratio and (not padding):
```
Comment (VictorHe-1) [inline deploy/py_infer/src/data_process/preprocess/transforms/det_transforms.py:22]: 函数默认值会报错，是不是不合理？

**Review by HaoyangLee — APPROVED** [review]

File: deploy/py_infer/src/data_process/preprocess/transforms/det_transforms.py (hunk 4 above)
Comment (liangxhao) [reply deploy/py_infer/src/data_process/preprocess/transforms/det_transforms.py:22]: 已修改，padding=True，不和基类一致，并给出了提示

**Pull request merged**